File: c/test/sqrt.c
#include <stdio.h>
#include <math.h>

int main(void)
{
    int x = 0;
    int m = 0;
    int n = 0;
    printf("start.\r\n");

    x = 1;
    while(x++) {
        m = (int)sqrt(x + 100);
        n = (int)sqrt(x + 100 + 168);

        if ((m * m == x + 100) && (n * n == x + 100 + 168)) {
            printf("The num is: %d\r\n", x);
            break;
        }
    }

    return 0;
}

File: python/sqrt.py
#!/usr/bin/python

import math

x = 1;
while x > 0:
    x = x + 1
    m = int(math.sqrt(x + 100))
    n = int(math.sqrt(x + 100 + 168))

    if m * m == x + 100 and n * n == x + 100 + 168:
        print(x)
        break

print"-------"

a = math.sqrt(2)
print"a == ", a

b = int(math.sqrt(2))
print"b == ", b

